src/textures/Texture.js

```diff
@@ -316,5 +316,6 @@ class Texture extends EventDispatcher {
 Texture.DEFAULT_IMAGE = null;
 Texture.DEFAULT_MAPPING = UVMapping;
 Texture.DEFAULT_ANISOTROPY = 1;
+Texture.DEFAULT_COLOR_SPACE = NoColorSpace;
```

Comment:
Setting the "default color space" to "not a color space" seems extremely odd to me.